src/LabelList.php

```diff
@@ -30,4 +34,20 @@ public static function create(array $itemList): LabelList
         }
         return $labelList;
     }
+
+    /**
+     * @return string
+     */
+    public function toString(): string
```

Comment:
What about to use the magic method `__toString` instead?

That would reduce some complexity. You do not need to check if $tagList is set 
(see https://github.com/jtl-software/goprometrics-client/pull/1/files#diff-12b9c865ef6e7b5c13e950ff283494faR41 and https://github.com/jtl-software/goprometrics-client/pull/1/files#diff-6d78098faa0ff25cdee7a498ed42bee6R33)